src/OpenTK.Mathematics/Geometry/Ray3.cs

```diff
@@ -0,0 +1,143 @@
+﻿//
+// Ray.cs
+//
+// Copyright (C) 2021 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+
+using System;
+using System.Diagnostics.Contracts;
+
+namespace OpenTK.Mathematics.Geometry {
```

Comment:
Please check what the formatting style of OpenTK is atm, I believe its `{` go on a new line